package.sh

```diff
@@ -1,11 +1,23 @@
 #!/bin/bash -ex
-export DEBIFY_IMAGE='registry.tld/conjurinc/debify:1.11.5'
+export DEBIFY_IMAGE='registry.tld/conjurinc/debify:2.0.0'
 
 docker run --rm $DEBIFY_IMAGE config script > docker-debify
 chmod +x docker-debify
 
+# Creates possum deb
 ./docker-debify package \
   --dockerfile=Dockerfile.fpm \
+  --output=deb \
   possum \
   -- \
   --depends tzdata
+  --force
```

Comment:
why did we add the `--force` flag?

Reply:
Misunderstood @rafis3 comment in slack about it. I will put -rpm-rpmbuild-define “_build_id_links none” instead.

Reply:
Changed it